src/Microsoft.TemplateEngine.Cli/TableOutput/HelpFormatter.cs

```diff
@@ -113,11 +113,11 @@ public string Layout()
                 {
                     for (int i = 0; i < _columns.Count - 1; ++i)
                     {
-                        b.Append(header[i].GetTextWithPadding(j, _columns[i].CalculatedWidth));
+                        b.Append(header[i].GetTextWithPadding(j, _columns[i].CalculatedWidth, _columns[i].RightAlign));
```

Comment:
Not exactly your change, but would it be easy to estimate the length of the string and use it in the constructor of `StringBuilder`?

Reply:
I would refrain from it at the moment: it is quite easy to estimate length using single line rows, but not for multiline rows. Multiline rows implementation is not used, however in general supported. I don't want to implement estimation not considering multi-line rows. 
Also we discussed that this class might be removed when we switch to new CommandLine library, so probably it's not worth to further invest in it.